Docs/Configuration.tex

```diff
@@ -6154,6 +6158,124 @@ \subsection{OpenRuntime}\label{uefiruntime}
+or to the closely related (but not identical)
+\href{https://fedoraproject.org/wiki/Changes/BootLoaderSpecByDefault}{systemd BootLoaderSpecByDefault}
+specification. In effect this means Linux distributions where the available boot options are found in
+\texttt{\{ESP\}/loader/entries/*.conf} files (for instance \texttt{/boot/efi/loader/entries/*.conf})
+or in \texttt{\{boot\}/loader/entries/*.conf} files (for instance \texttt{/boot/loader/entries/*.conf}).
+This applies to most Fedora-related distros including Fedora itself, RHEL and variants.
+
+For distributions which do not use the above scheme, \texttt{OpenLinuxBoot} will also autodetect and
+boot \texttt{\{boot\}/vmlinuz*} kernel files directly, after linking these automatically (based on the
+kernel version in the filename) to their associated \texttt{\{boot\}/init*} ramdisk files. This applies
+to most Debian-related distros, including Debian itself, Ubuntu and variants.
+
+In both cases, the method of starting the kernel relies on it being compiled with
+EFISTUB, however this applies to almost all modern distros, particularly those which use systemd.
+
+The latest kernel version of a given install is always shown. Additional versions, recovery versions,
+etc. are added as auxiliary boot entries, so depending on OpenCore's \texttt{HideAuxiliary} setting
+may not be shown until the space key is pressed.
+
+By default most Linux distributions keep their boot files on an EXT4 file system (even
+if the main system partition is something else such as BTRFS), therefore a suitable UEFI EXT4 file system
+driver is required. Amongst other sources, a suitable driver \texttt{ext4\_x64.efi} can be
+found within the latest \href{https://sourceforge.net/projects/refind/files/latest/download}{rEFInd}
+distribution. Pure Boot Loader Spec keeps all boot files on the EFI System Partition (or an extended
+bootloader partition), therefore requires no additional driver - but this is not widely used. Similarly,
+a BTRFS driver will be required if you have a less standard setup where your boot files are on a BTRFS
+partition.
+
+The default parameter values should work well, but if you need to parameterise this driver the following
+options may be specified in \texttt{UEFI/Drivers/Params}:
+
+\begin{itemize}
+  \tightlist
+	\item \texttt{flags} - Default: all flags except \texttt{LINUX\_BOOT\_ADD\_DEBUG\_INFO} are set. \medskip
+      
+	Available flags are: \medskip
+      
+  \begin{itemize}
+  \tightlist
+    \item \texttt{0x00000001} (bit \texttt{0}) --- \texttt{LINUX\_BOOT\_SCAN\_ESP},
+    Allows scanning for entries on EFI System Partition.
+    \item \texttt{0x00000002} (bit \texttt{1}) --- \texttt{LINUX\_BOOT\_SCAN\_XBOOTLDR},
+    Allows scanning for entries on Extended Bootloader partition.
+	  \item \texttt{0x00000004} (bit \texttt{2}) --- \texttt{LINUX\_BOOT\_SCAN\_EXT4},
+	  Allows scanning for entries on EXT4, and on other file systems (e.g. BTRFS)
```

Comment:
ScanPolicy is done by file system type GUID, so I believe cannot distinguish EXT4 and BTRFS as it currently works, since both are marked as Linux filesystem `mLinuxFileSystemPartitionTypeGuid`.

Reply:
Maybe make it clear then that it accepts both?